algoliasearch_django/models.py

```diff
@@ -80,10 +80,7 @@ def __init__(self, model, client, settings):
         self.__named_fields = {}
         self.__translate_fields = {}
 
-        try:
-            all_model_fields = model._meta.get_all_field_names()
-        except AttributeError:  # get_all_field_names is removed in Django >= 1.10
-            all_model_fields = [f.name for f in model._meta.get_fields()]
+        all_model_fields = [f.name for f in model._meta.get_fields() if not f.is_relation]
```

Comment:
I would use `_meta.local_fields` instead on Django < 1.8 :

```diff
diff --git a/algoliasearch_django/models.py b/algoliasearch_django/models.py
index 46eae60..f9d7caf 100644
--- a/algoliasearch_django/models.py
+++ b/algoliasearch_django/models.py
@@ -80,7 +80,10 @@ class AlgoliaIndex(object):
         self.__named_fields = {}
         self.__translate_fields = {}

-        all_model_fields = [f.name for f in model._meta.get_fields() if not f.is_relation]
+        try:
+            all_model_fields = [f.name for f in model._meta.get_fields() if not f.is_relation]
+        except AttributeError: # get_fields requires Django >= 1.8
+            all_model_fields = [f.name for f in model._meta.local_fields]
```

Reply:
Hi @PLNech whoops, no I didn't run the tests using `tox`. I've never used it before so it totally slipped my mind when running the tests. I'll make these changes some time today! Thanks for the review.

Reply:
Noprob, you're welcome!

Reply:
Hey @johngriebel, do you want to move forward with this or should I apply these changes on top of your contribution and merge? 🙂

Reply:
@PLNech Hey, my apologies for letting this sit so long. My trial key with Algolia expired, then in the meantime my day job has gotten hectic; you know how it goes. Anyhow, go ahead and apply your changes on top of mine!

Reply:
Hey, no need to apologise. I definitely can relate to this!

I'll go with these changes and merge then, thanks again for the report and contribution 👍